contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,107 @@
+            artist_url = album['artists'][0]['external_urls']['spotify']
+            return [album_name, album_url, artist_name, artist_url]
+
+        def track_content(track):
+            song_name = track['name']
+            song_url = track['external_urls']['spotify']
+            album_name = track['album']['name']
+            album_url = track['album']['external_urls']['spotify']
+            artist_name = track['artists'][0]['name']
+            artist_url = track['artists'][0]['external_urls']['spotify']
+            return [song_name, song_url, album_name, album_url, artist_name, artist_url]
+
+        base_url = 'https://api.spotify.com/v1/search?q='
+        original_content = message['content']
+        original_content = original_content.replace('@spotify ', '')
+        new_content = ''
+
+        if original_content.split(' ', 1)[0] == "album":
+            new_content += " Here are the albums with \"%s\" in them: " % (original_content[6:])
+            encoded_message = urllib.parse.quote(original_content[6:])
+            url = base_url+encoded_message+"&type=album"+"&limit=10"
+            raw_data = urllib.request.urlopen(url).read()
+            raw_albums_list = json.loads(raw_data)['albums']['items']
+            if raw_albums_list != []:
+                for album in raw_albums_list:
+                    content = album_content(album)
+                    new_content += "\n [%s](%s) by [%s](%s)\n" % tuple(content)
+            else:
+                new_content = "Sorry, we couldn't find anything with \"%s\"." % (original_content[6:])
+        elif original_content.split(' ')[0] == "artist":
+            new_content += " Here the artists with \"%s\" in them: " % (original_content[7:])
+            encoded_message = urllib.parse.quote(original_content[7:])
+            url = base_url+encoded_message+"&type=artist"+"&limit=10"
+            raw_data = urllib.request.urlopen(url).read()
+            json_data = json.loads(raw_data)['artists']['items']
+            if json_data != []:
+                for artist in json_data:
+                    content = artist_content(artist)
+                    new_content += "\n [%s](%s) \n" % tuple(content)
+            else:
+                new_content = "Sorry, we couldn't find anything with \"%s\"." % (original_content[7:])
+        elif (original_content.split(' ')[0] == "song"):
+            new_content += " Here are tracks with \"%s\" in them: " % (original_content[5:])
+            encoded_message = urllib.parse.quote(original_content[5:])
+            url = base_url+encoded_message+"&type=track"+"&limit=10"
+            raw_data = urllib.request.urlopen(url).read()
+            json_data = json.loads(raw_data)['tracks']['items']
+            if json_data != []:
+                for track in json_data:
+                    content = track_content(track)
+                    new_content += "\n [%s](%s) from the album [%s](%s) by [%s](%s)\n" % tuple(content)
```

Comment:
This template looks hardcoded in multiple places. Could you please move it to some const?